docs/osx/el_capitan_10.11.2_install_notes.md

```diff
@@ -0,0 +1,87 @@
+WebSiteOne install
+————————
+
+Fresh 10.11.2 (El Capitan) OSX
+
+1) Using git for the first time prompted installing developer tools
+
+2) pull down code
+
+```
+git clone http://github.com/AgileVentures/WebsiteOne
```

Comment:
@tansaku I added a note at the top of this document. If that's fine we can merge it. As we progress with the documentation, we should have a doc that covers common issues during setup. Then we extract the ones you have here with solutions into that document.